test/core/run.py

```diff
@@ -22,10 +22,14 @@
 arguments = parser.parse_args()
 sys.argv = sys.argv[:1]
 
+mvp_test_files = glob.glob(os.path.join(inputDir, "*.wast"))
```

Comment:
Nit: "mvp" is a bit misleading here, since the dir also contains post-mvp tests at this point. Maybe rather "main"?

Reply:
Good point, done!